examples/step-12/step-12.cc

```diff
@@ -209,20 +208,15 @@ namespace Step12
     // fact other arguments already bound.
     static void integrate_cell_term(DoFInfo &dinfo, CellInfo &info);
     static void integrate_boundary_term(DoFInfo &dinfo, CellInfo &info);
-    static void integrate_face_term(DoFInfo & dinfo1,
-                                    DoFInfo & dinfo2,
-                                    CellInfo &info1,
-                                    CellInfo &info2);
+    static void
+    integrate_face_term(DoFInfo &dinfo1, DoFInfo &dinfo2, CellInfo &info1, CellInfo &info2);
   };
 
 
   // We start with the constructor. The 1 in the constructor call of
   // <code>fe</code> is the polynomial degree.
   template <int dim>
-  AdvectionProblem<dim>::AdvectionProblem() :
-    mapping(),
-    fe(1),
-    dof_handler(triangulation)
+  AdvectionProblem<dim>::AdvectionProblem() : mapping(), fe(1), dof_handler(triangulation)
```

Comment:
:-(

Reply:
I agree. This is incredibly ugly.

What about we change the configuration:
```
 BreakBeforeBraces: Custom
 BreakBeforeInheritanceComma: false
 BreakBeforeTernaryOperators: false
-BreakConstructorInitializers: AfterColon
+BreakConstructorInitializersBeforeComma: true
 BreakStringLiterals: false
 
 ColumnLimit: 80
 
 CompactNamespaces: false
 
-ConstructorInitializerAllOnOneLineOrOnePerLine: true
 ConstructorInitializerIndentWidth: 2
 
 ContinuationIndentWidth: 2
```

To get the following result:
```
// We start with the constructor. The 1 in the constructor call of
// <code>fe</code> is the polynomial degree.
template <int dim>
AdvectionProblem<dim>::AdvectionProblem()
  : mapping()
  , fe(1)
  , dof_handler(triangulation)
{}

```

Reply:
I must say that I do like the symmetry in this variant through comma and colon, even though I intrinsically would have very much preferred the comma at the end of the line otherwise...

Reply:
`BreakConstructorInitializers: BeforeComma` seems to be the only option that enforces a new line breaks here.

Reply:
#6696 fixes this